src/main/java/com/minecolonies/coremod/entity/ai/citizen/deliveryman/EntityAIWorkDeliveryman.java

```diff
@@ -134,14 +134,14 @@ public AIState gather()
     {
         if (maximalGatherCount < 0)
         {
-            maximalGatherCount = Configurations.requestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.requestSystem.maximalBuildingsToGather - Configurations.requestSystem.minimalBuildingsToGather));
+            maximalGatherCount = Configurations.RequestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.RequestSystem.maximalBuildingsToGather - Configurations.RequestSystem.minimalBuildingsToGather));
         }
 
         if (gatherTarget == null)
         {
             if (gatherCount == maximalGatherCount)
             {
-                maximalGatherCount = Configurations.requestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.requestSystem.maximalBuildingsToGather - Configurations.requestSystem.minimalBuildingsToGather));
+                maximalGatherCount = Configurations.RequestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.RequestSystem.maximalBuildingsToGather - Configurations.RequestSystem.minimalBuildingsToGather));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 244 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)